lib/danger/scm_source/git_repo.rb

```diff
@@ -4,20 +4,25 @@
 
 module Danger
   class GitRepo
-    attr_accessor :diff, :log
+    attr_accessor :diff, :log, :folder
 
     def diff_for_folder(folder, from: "master", to: "HEAD")
-      repo = Git.open folder
+      self.folder = folder
+      repo = Git.open self.folder
 
       merge_base = repo.merge_base(from, to)
+      ensure_merge_base_exists!(merge_base)
```

Comment:
Isn't it this commit (https://github.com/danger/danger.systems/commit/3e956f142ee4f9b5da9fb26f0c2e041b199c6c7a)? It looks like it would have been the HEAD of master at the time that that CI build was running.
I suspect that the test for the presence of the commit that `git merge-base` tells us about can be omitted, as I'd be quite surprised if git was able to produce a merge base that it does not have details of.